libs/core/src/components/sage-tab/sage-tab.tsx

```diff
@@ -0,0 +1,83 @@
+import { Component, Element, Host, h, Prop, Event, EventEmitter } from '@stencil/core';
+@Component({
+  tag: 'sage-tab',
+  styleUrl: 'sage-tab.scss',
+  shadow: false,
+})
+export class SageTab {
+  @Element() el: HTMLElement;
+
+  /**
+   * Sets the related tab name, this name must match a `sage-tabpanel`'s tab property, required
+  */
+  @Prop() tab: string;
+
+  /**
+   * Keeps track of the activeTab, this property is passed in by parent component
+  */
+  @Prop({mutable: true}) activeTab: string;
+
+  /**
+   * Keeps track of the parentComponent unique id, this property is passed in by parent component
+  */
+  @Prop({mutable: true}) parentComponent: string;
+
+  /**
+   * Keeps track of if the expected tab variant, this property is passed in by parent component
+  */
+  @Prop() variant: string;
+
+  /**
+   * Keeps track of if the tabpanel is selected, this property is computed on `componentWillUpdate()`
+  */
+  @Prop({mutable: true}) selected = false;
+ 
+  /**
+   * Emits an event upon tab click for `sage-tab` and `sage-tabpanel` to listen for
+   */
+  @Event({composed: false}) tabClick: EventEmitter<object>;
```

Comment:
Why did you elect to set `composed` to false?

Reply:
This is no longer needed. The `composed: false` is supposed to scope an event to the nearest shadow dom edge but in practice I found this doesn't work for our use case. I dug into this but came up empty handed. To solve this issue I've passed the parentCompomentID as well to make sure we can scope our listeners for these events instead.